integration-tests/templates/negative/_Cargo_no_abi_feature.toml

```diff
@@ -7,7 +7,9 @@ edition = "2021"
 crate-type = ["cdylib"]
 
 [dependencies]
-near-sdk = { default-features = false, version = "::sdk-version::" }
+# NOTE: uncomment line 11 and comment 12, if it's desirable to switch to published `near-sdk` version
+# near-sdk = { default-features = false, version = "::sdk-version::" }
```

Comment:
Should it be this way?

```suggestion
# near-sdk = { default-features = false, ::sdk-version:: }
```

Reply:
nope, `sdk-version` is `4.1.1`,
and `sdk-version-toml` is `version = "4.1.1"`

Reply:
@dj8yfo Do we need two ways of achieving the same thing? In other files I see we use `sdk-version-toml`:

https://github.com/near/cargo-near/pull/109/files#diff-ee562e36f9098ae09c28cb061037a3122c4492b7333f6ffb188b40efa01ba9d9R11

Reply:
@frol i'll replace with `sdk-version-toml` everywhere to reduce the number of variants to express the same thing